specs/features/Inconsistencies.md

```diff
@@ -0,0 +1,126 @@
+# Inconsistencies in Adaptive Cards platforms
+
+## Motivation
+
+Adaptive Cards is currently available on these platforms:
+
+* [UWP](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/uwp/getting-started) (Available in all WinRT-projected languages; renders using XAML)
+* .Net (Available in all .Net languages; renders using [HTML](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/net-html/getting-started) or [WPF](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/net-wpf/getting-started))
+* [Android](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/android/getting-started) (Java; renders using native Android platform)
+* [iOS](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/ios/getting-started) (ObjC; renders using native iOS platform)
+* [Javascript](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/getting-started) (renders using HTML/JS/CSS)
+* [ReactNative](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/react-native/getting-started) (community renderer)
+* Flutter (community renderer)
+
+This list of platforms represents at least 7 distinct implementations of Adaptive Cards at the rendering level. At each decision point along the way during implementation, the Adaptive Cards developer (i.e. someone writing code to parse or render Adaptive Cards) risks implementing behavior that diverges from intended behavior. One of the biggest selling points for Adaptive Cards is our ability to render across our supported platforms in a way that at once feels native to the supported platform while also having a look and feel that's representative of the card author's intent. Indeed, portability is one of our [core goals](https://docs.microsoft.com/en-us/adaptive-cards/#goals). Each time an inconsistency sneaks into the platform, it risks introducing costs to our partners. These costs have real consequences for the adoption of Adaptive Cards in the market:
```

Comment:
It'd be nice to list out specifically teh goals and non-goals of the work you are setting up to do in this spec to scope it a bit. For eg we are not attempting to solve the inconsistencies that occur across iOS and Android renderers being customized by a given partner. In addition the specific differences between Android and iOS in particular (so the intra-mobile) inconsistencies need not be tackled here; They are being driven by Joseph (@nesalang) and perhaps can be referenced in an Appendix or References section here but clarify the scope of this document doesnt address that category. ANd so on..